torchvision/csrc/io/image/cpu/decode_jpeg.cpp

```diff
@@ -70,6 +70,7 @@ static void torch_jpeg_set_source_mgr(
 } // namespace
 
 torch::Tensor decode_jpeg(const torch::Tensor& data, ImageReadMode mode) {
+  C10_LOG_API_USAGE_ONCE("torchvision.io.decode_jpeg");
```

Comment:
Do we want to use the same log message as Python's `decode_jpeg` here? Or we want to have a separate distinct message?

Reply:
It depends on whether we're particular interested in events from C++ API calls. Our logging system currently couldn't tell whether it's logged in C++ or Python.

I can add "_cpp" suffix here (e.g. `torchvision.io.decode_jpeg_cpp`) for now, we could always further process in log pipeline.